cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc

```diff
@@ -0,0 +1,13 @@
+[id="run-gitops-control-plane-workload-on-infra-nodes"]
+= Running GitOps control plane workload on infra nodes
+:context: run-gitops-control-plane-workload-on-infra-nodes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Infrastructure nodes allow customers to isolate infrastructure workloads for two primary purposes:
```

Comment:
```suggestion
You can use infrastructure nodes to isolate infrastructure workloads for two primary purposes:
```
Per IBM SG: 'Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."'